test/box/graceful_shutdown.result

```diff
@@ -0,0 +1,251 @@
+-- test-run result file version 2
```

Comment:
I want to pay attention, that your test fails https://github.com/tarantool/tarantool/runs/2456579467. So i think
it is necessary to understand why!